src/orange/OrangeData.hh

```diff
@@ -30,7 +30,7 @@ namespace celeritas
  */
 struct OrangeParamsScalars
 {
-    size_type max_level{};
+    size_type max_num_levels{};
```

Comment:
I'd add a comment about this (e.g., it's 1 for non-nested geometry, it's the depth of the universe tree DAG). It's also equivalent to the `max_depth` of the vecgeom implementation: maybe `max_depth` is a better name here? Especially because we have `max_{blah}` below this, so `max_num` is inconsistent.